exporter/opentelemetry-exporter-otlp-proto-grpc/tests/test_otlp_trace_exporter.py

```diff
@@ -460,24 +459,9 @@ def test_otlp_headers(self, mock_ssl_channel, mock_secure):
             (("user-agent", "OTel-OTLP-Exporter-Python/" + __version__),),
         )
 
-    @patch("opentelemetry.exporter.otlp.proto.grpc.exporter.backoff")
-    @patch("opentelemetry.exporter.otlp.proto.grpc.exporter.sleep")
-    def test_handles_backoff_v2_api(self, mock_sleep, mock_backoff):
```

Comment:
Just realized I need to re-add this somewhere. Will update this PR with that change.